renderdoc/driver/d3d11/d3d11_counters.cpp

```diff
@@ -332,6 +348,73 @@ void D3D11Replay::FillTimersAMD(uint32_t &eventStartID, uint32_t &sampleIndex,
   }
 }
 
+static void FlushGPU(WrappedID3D11Device *pDevice, WrappedID3D11DeviceContext *pImmediateContext)
```

Comment:
Can this function be deleted and instead have code use the `D3D11Replay::SerializeImmediateContext` a bit above instead?